daemon_config_linux_test.go

```diff
@@ -128,7 +128,7 @@ func TestDaemonRuntimeRoot(t *testing.T) {
 	}()
 	configTOML := `
 [plugins]
- [plugins.cri]
+ [plugins."io.containerd.grpc.v1.cri"]
```

Comment:
just curiosity, the `"` is required here?

Reply:
you need to quote things in toml for the parser as the `.` notation means something already.